lib/safer_rails_console/patches/sandbox/auto_rollback.rb

```diff
@@ -25,7 +25,7 @@ def self.handle_and_reraise_exception(e)
         module ActiveRecord
           module ConnectionAdapters
             module PostgreSQLAdapter5
-              def execute_and_clear(sql, name, binds, prepare: false)
```

Comment:
I think to better handle the kwargs changes in ruby 3+, I would use `...` instead of `*`. We recently updated this to only support ruby 2.7+.